### ISSUE TITLE
<webview> CSS display changes

### ISSUE THREAD

**Issue by bigtimebuddy** [electron/electron#5059]
* Electron version: 0.37.4
* Operating system: OS X 10.11.3

I'm having an issue where `<webview>` elements are behaving differently interacting with CSS then in 0.36 versions of Electron. The element is cropped when they have a CSS `display` property of `block` or `inline-block`. It must be set to `flex` or else the content is cropped off. Is this intended?

### Example

**index.js**
```js
"use strict";

// Arguments: src, width, height, background, title
const electron = require('electron');
const app = electron.app;
const ipc = electron.ipcMain;
const Menu = electron.Menu;
const BrowserWindow = electron.BrowserWindow;

// Window to preview in
let mainWindow = null;

// Quit when all windows are closed. 
app.on('window-all-closed', function() { 
    app.quit(); 
});

// This method will be called when Electron has finished 
// initialization and is ready to create browser windows. 
app.on('ready', function() {

    mainWindow = new BrowserWindow({
        width: 550,
        height: 400,
        useContentSize: true,
        center: true,
        skipTaskbar: true,
        show: true
    });

    mainWindow.webContents.toggleDevTools();
    mainWindow.loadURL('file://' + __dirname + '/index.html'); 
    mainWindow.on('closed', function() { 
        mainWindow = null; 
    });
});
```

**index.html**

```html
<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <meta name='viewport' content='width=device-width, initial-scale=1.0'>
        <style>
            #view {
                display:block;
                width:100%;
                height:100%;
                position:absolute;
                border:1px solid red; 
                box-sizing:border-box;
            }
            body {
                margin:0;
            }
        </style>
    </head>
    <body>
        <webview id="view" src="https://github.com/"></webview>
    </body>
</html>
```
**Preview**

![screen shot 2016-04-06 at 4 58 51 pm](https://cloud.githubusercontent.com/assets/864393/14332728/e05ec4e6-fc18-11e5-8e3a-13ec8ae21ff0.png)

**Comment by MarshallOfSound** [comment]
I needed to add this CSS to handle this change.  (Technically this is LESS but whatever :laughing: )

```less
webview {
    &::shadow {
        object {
            height: 100%;
            width: 100%;
            outline: 0;
        }
    }
}
```

**Comment by christoth** [comment]
Yes, this was intended. Please use display:flex & display:flex-inline. It should be have the same with fixed sizing as electron version < 0.36.10
The change to use a flex based layout was made to support embedding webview into flexbox layout, because the child object was not properly filling the webview parent to full width & height.
I need to update the doc this week.

**Comment by bigtimebuddy** [comment]
@christoth Thanks for the info. Updated docs would definitely be nice, but makes sense.
